src/app/component/core/header-new/header-new.component.html

```diff
@@ -61,7 +61,7 @@
                   </div>
         <div class="navigation-menu-right">
               <ul>
-                          <li class="search"><a  href="#">    <img src="assets/img/icon/search.png" alt="search">  </a></li>
+                          <li class="search"><a  href="#" onclick = "return false" (click)="toggleSearchPage()">    <img src="assets/img/icon/search.png" alt="search">  </a></li>
```

Comment:
Please, write attributes in separate rows

Reply:
Fixed